Wasm/Bytes.lean

```diff
@@ -50,9 +52,9 @@ def extractTypes (m : Module) : ByteArray :=
     -- TODO: check if we support > 255 params. Do the same for each length and size entries!
     let header := ByteArray.mk #[0x60, params.length.toUInt8]
```

Comment:
Wdym? Honestly I have no idea what much of the binary output does, so I'll gladly accept your help with checking. But I think we can postpone it until we implement proper tests, it's unsustainable to check by hand.

Reply:
Auto complete. We should LEB the length. Same for the other PR.

Reply:
I don't understand what exactly we should do here, what LEBbing the length does, and when it should be done. There's so little documentation! :/

Reply:
Well, LEB allows us to encode an int of any size in a compact way. It's like UTF to ASCII. 1 is still `1`, but bigger things are represented in a scaling encoding.

So whenever you need to count something, you need to use LEB, because if the amount of that thing is greater than 255 our implementation will explode. Which isn't a desired behaviour :D

Reply:
Great explanation! I added `uLeb128` to all lengths I could find, please recheck.